docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -57,6 +57,7 @@ Documentation
 
 Requirements
 ~~~~~~~~~~~~
+* minimum pandas version increased to v0.22.0, released Dec 31, 2017. (:pull:`1003`)
```

Comment:
```suggestion
* Minimum pandas version increased to v0.22.0, released Dec 31, 2017. (:pull:`1003`)
```

Reply:
I like the inclusion of the release date!

Reply:
Just copying @mikofski :)  https://github.com/pvlib/pvlib-python/blame/master/docs/sphinx/source/whatsnew/v0.7.0.rst#L214